pages/apim/3.x/v4-beta/v4-beta-event-native-apim-introduction.adoc

```diff
@@ -0,0 +1,83 @@
+
+=== WebSocket and SSE entrypoints
+
+The Server-Sent Events (SSE) and WebSocket entrypoints are available as separate plugins you can download from the link:https://download.gravitee.io/#graviteeio-apim/plugins/entrypoints/[entrypoints plugin download page]. Please note that you need an Enterprise Edition (EE) license to use the SSE Advanced entrypoint capabilities. 
+
+This feature enables you to front your event-driven backend - currently only using Kafka or the MQTT5 connectors  - with Websocket or Server-Sent Events (SSE) so that API consumers can access events from the Kafka backend via one of these more consumer-friendly protocols.
+
+For example, if an API consumer cannot connect to a native Kafka instance due to technical limitations (for example, if they cannot implement a Kafka client themselves), Kafka topics can be made available to that consumer via a more consumer-friendly protocol such as SSE or Websocket.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-event-consumption-streaming.png %}[]
+
+This feature enables you to potentially cover use cases in the fields of Event-Driven Architecture (EDA) and Internet of Things (IoT) without the Gateway and API Management components becoming a bottleneck.
+
+NOTE: In comparison to the link:#http_post_and_http_get_entrypoints[HTTP POST entrypoint use case] using Kafka (where the HTTP POST method is just about sending information from the client app to Kafka), with Websocket it is possible for the client app to send and receive information from Kafka as well. SSE can consume information from Kafka.
+
+=== Webhook entrypoint
+
+This is a feature that enables you to front your backend with a link:https://en.wikipedia.org/wiki/Webhook[Webhook^]. It has CE/OSS and EE versions - Webhook, and Webhook Advanced, respectively. Both variants are available as part of a separate plugin you can download from the link:https://download.gravitee.io/#graviteeio-apim/plugins/entrypoints/[entrypoints plugin download page]. Please note that you need an Enterprise Edition (EE) license to use the Webhook Advanced entrypoint capabilities. 
+
+With Webhook support, an API consumer subscribes to the Webhook API, which basically indicates to the Gateway to listen for specific Kafka messages and then call the consumer to let them know that an expected event has occurred. This communication is performed via Webhook, however it is always up to the Gateway to push the events or messages to the consumer.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-event-consumption-webhook.png %}[]
+
+=== Security policies
```

Comment:
Previous sections were talking about the different entrypoints supported and then it switches to "Security policies" which appears not really related.

Reply:
These are all features that are under the heading "BETA features". Arguably, the security policies are also beta features, but because they are the same as in previous releases, I will move this heading 'up' one level. This will make it a bit clearer.

Reply:
Ok but I'm still not convinced sorry. Don't you think that instead of talking about security policies we should more talk about plans ? Plan are the way to secure apis. Under the hood, plan are relying on security policy implementations but this is a technical point of view. When a user wants to secure his api, he creates a plan and select a security type (which will technically use the corresponding policy) but he never put a "security policy" by himself.

Reply:
OK. I think I understand now. Would it make more sense to remove this section entirely? It seems that it works the same way as in previous versions anyway.

Reply:
OK, how does this sound?

== Security

Gravitee can still provide the same security as in previous versions. You can secure access using an API key, JWT, or OAuth.